staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width.go

```diff
@@ -83,6 +85,11 @@ func (e WorkEstimatorFunc) EstimateWork(r *http.Request, flowSchemaName, priorit
 	return e(r, flowSchemaName, priorityLevelName)
 }
 
+type WorkEstimator interface {
+	Estimate(*http.Request, string, string) WorkEstimate
+	MaximumSeatRatio(WorkEstimate) float64
```

Comment:
The name of this method is nicely short but a bit awkward.  A "ratio" is between two values, but this method name identifies only one of them.  I recommend a comment that identifies the denominator.

Reply:
I agree and added some comments but... I don't know any better name for `MaximumSeatRatio`. I could use some advice. ;)

Reply:
First one minor comment: I think "seats" works better here than "seat", because we are not trying to talk about one singular seat.

Here is one thought.  The word "relative" is sometimes used to talk about a ratio with the implication that the denominator is understood but not mentioned.  For example, a "relative error" is the error divided by the normal/expected/good value.  How about calling this "RelativeMaximumSeats"?

Reply:
I like `RelativeMaximumSeats`. Used that for the rename. I'd consider having maybe `RelativeToMaximumSeats`?

Reply:
Ah, well, there are two sorts of maximum going on here.  One is the bound that is configured.  The other maximum is between the two phases of execution (regular, then the dally to account for WATCH notifications).  The latter is the sense of the word "maximum" appearing here.